docs/framework/install/reference-assemblies.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Build apps against Microsoft.NETFramework.ReferenceAssemblies"
+description: "Developers can target .NET Framework by referencing the official reference assemblies NuGet package."
+ms.date: 11/25/2022
+helpviewer_keywords:
+  - ".NET Framework, targeting"
+---
+# Build apps against Microsoft.NETFramework.ReferenceAssemblies
+
+When you target a particular version of .NET Framework, by default your application is built by using the reference assemblies that are included with that version's developer pack. In scenarios where the matching developer pack cannot be installed on the computer, you can alternatively build against reference assemblies distributed via a NuGet package.
+
+## Update project files
+
+Each project that should build against the reference assemblies NuGet package needs to include the following package reference.
+
+```xml
+<PropertyGroup>
+  <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.3" PrivateAssets="All" />
```

Comment:
Doesn't the .NET SDK already automatically add this reference? I don't believe there should be a need for customers to ever need to do this themselves.

Reply:
Indeed, SDK-based projects should be getting the reference by default. I am primarily interested in non-SDK-style scenarios where the user would have to add the reference explicitly, just because that's what most legacy projects use. It looks like the guidance should be different for SDK and non-SDK here. Thank you!